docs/tutorial/part-one/index.md

```diff
@@ -198,191 +198,190 @@ export default () => (
 )
 ```
 
-![Adding Header component](06-header-component.png)
+![Toevoegen van een Header component](06-header-component.png)
 
-In the browser, the “About Gatsby” header text should now be replaced with “This is a header.” But you don’t want the “About” page to say “This is a header.” You want it to say, “About Gatsby”.
+In de browser moet de koptekst "Over Gatsby" nu zijn vervangen door "Dit is een koptekst." Maar je wilt niet dat de "Over"-pagina "Dit is een koptekst" laat zien. Je wilt dat er "Over Gatsby” wordt getoond.
 
-4.  Head back to `src/components/header.js` and make the following change:
+4.  Ga terug naar `src/components/header.js` en doe de volgende aanpassing:
 
 ```jsx:title=src/components/header.js
 import React from "react"
 
 export default props => <h1>{props.headerText}</h1> {/* highlight-line */}
 ```
 
-5.  Head back to `src/pages/about.js` and make the following change:
+5.  Ga terug naar `src/pages/over.js` en doe de volgende aanpassing:
 
-```jsx:title=src/pages/about.js
+```jsx:title=src/pages/over.js
 import React from "react"
-import Header from "../components/header"
+import Header from "../components/header";
 
 export default () => (
   <div style={{ color: `teal` }}>
-    <Header headerText="About Gatsby" /> {/* highlight-line */}
+    <Header headerText="Over Gatsby" /> {/* highlight-line */}
     <p>Such wow. Very React.</p>
   </div>
 )
 ```
 
-![Passing data to header](07-pass-data-header.png)
+![Data doorgeven aan de koptekst](07-pass-data-header.png)
 
-You should now see your “About Gatsby” header text again!
+Je zou nu de "Over Gatsby" koptekst opnieuw moeten zien!
 
-### What are “props”?
+### Wat zijn “props”?
 
-Earlier you defined React components as reusable pieces of code describing a UI. To make these reusable pieces dynamic you need to be able to supply them with different data. You do that with input called “props". Props are (appropriately enough) properties supplied to React components.
+Eerder definieerde je React-componenten als herbruikbare stukjes code die een gebruikersinterface beschrijven. Om deze herbruikbare onderdelen dynamisch te maken, moet je ze van verschillende gegevens kunnen voorzien. Je doet dat met invoer met de naam "props". Props zijn (voldoende) eigenschappen die aan React-componenten worden geleverd.
```

Comment:
"(voldoende)" doesn't seem to make sense here because in Dutch the link between props and properties is not there.

I would also suggest to use "parameters" instead of "eigenschappen".